app/component/map/MapWithTracking.js

```diff
@@ -376,13 +376,21 @@ class MapWithTrackingStateHandler extends React.Component {
         onSelectLocation={this.props.onSelectLocation}
         leafletEvents={{
           onDragstart: () => {
-            this.disableMapTracking();
-            this.setState({ humanIsScrolling: true });
+            if (this.isCanceled) {
+              this.disableMapTracking();
+              this.setState({ humanIsScrolling: true });
+            }
+          },
+          onMoveend: () => {
+            if (this.isCanceled) {
+              this.setState({ keepOnTracking: false });
+            }
           },
-          onMoveend: () => this.setState({ keepOnTracking: false }),
           onDragend: () => {
-            this.updateCurrentBounds();
-            this.setState({ humanIsScrolling: false });
+            if (this.isCanceled) {
+              this.updateCurrentBounds();
+              this.setState({ humanIsScrolling: false });
```

Comment:
This caused the crash, because state was changed when the component was already unmounted. This condition here is quite dirty and there should be a better solution of avoiding this, so let me know if you have one.

Reply:
I don't have one. I will merge this in as is. If you have a better idea, we can change it later.